### PR TITLE
Rename to performance to perf to match top-level test target

### PR DESCRIPTION
**Pull request by smlambert**
- Related to #980 
- Remove odm 

Signed-off-by: smlambert <slambert@gmail.com>

### CODE REVIEW

**Comment by smlambert** [comment]
Grinder (BUILD_LIST=perf, TARGET=akka-uct): https://ci.adoptopenjdk.net/view/all/job/Grinder_Sandbox/431/

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -406,9 +406,9 @@ def post(output_name) {
 					uploadToArtifactory()
 				}
 			}
-			//for performance test, achive regardless the build result
+			//for performance test, archive regardless the build result
 			//TODO: Need to soft-code the output dir path
-			def benchmark_test_output_dir = 'jvmtest/performance/odm/ilog_wodm881/leftoverResults';
+			def benchmark_test_output_dir = 'jvmtest/perf/odm/ilog_wodm881/leftoverResults';
 			if (fileExists(benchmark_test_output_dir)) {
```
Comment (sophia-guo) [inline buildenv/jenkins/JenkinsfileBase]: If odm is removed this archive for odm specific could also be removed?

**Comment by sophia-guo** [comment]
I also see some liberty files are removed too. Just to confirm if that is expected? @smlambert

File: buildenv/jenkins/JenkinsfileBase (hunk 1 above)
Comment (smlambert) [reply buildenv/jenkins/JenkinsfileBase]: Ah yes, ideally, we replace that with an archiving for all perf targets that get run.  Depending on how involved that is, I may defer to an upcoming PR (I will have another one prepped for later this week).

**Comment by smlambert** [comment]
I did not intend to remove files so will correct and push changes shortly. I think the confusion is that those files are in a folder named bin, which is normally houses files that are built at execution time and not stored in repos (so in my gitignore list).

**Comment by sophia-guo** [comment]
Perf test results may not be under TestConfig/test_output_*. The output location may be different we may need to define case by case later. That will be done later. Merge this first.

**Comment by smlambert** [comment]
Thanks @sophia-guo - for the record, I ran a Grinder, and the json output from akka-uct benchmark is now archived as part of the tar.gz file (as I had hoped it would be), https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_SandboxExtra/2/artifact/

**Pull request merged**